lavalink/src/model.rs

```diff
@@ -627,12 +632,32 @@ pub mod incoming {
         /// The base64 track that was affected.
         pub track: String,
     }
+
+    /// The voice websocket connection to Discord has been closed.
+    #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+    #[non_exhaustive]
+    #[serde(rename_all = "camelCase")]
+    pub struct WebsocketClosed {
+        /// The guild ID of the player.
```

Comment:
Prefer not to prefix with `The` when they all start with it.

Reply:
This is also true for the other events, should those be updated too?

Reply:
Out of scope for this PR but at some point there should be another PR to address this